### PR TITLE
docs(drawer): add drawer props desc

### PR DESCRIPTION
**Pull request by Otto-J**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by YunYouJun** [comment]
Thanks. Could you add them for other languages, Spanish/Japanese, etc. in English first?

> Or just translate theme by deepl? 🤣

**Comment by Otto-J** [comment]
ok

**Comment by Otto-J** [comment]
搞定了，补充了五个遗漏的属性，多语言版本也弄好了。注意到部分文档没有及时trim 空格，这个规则留意下。

File: website/docs/zh-CN/drawer.md [hunk 1]
```diff
@@ -431,6 +435,7 @@ Drawer has almost identical attributes as Dialog.
 | title     | Drawer 的标题，也可通过具名 slot （见下表）传入 | string    | — | — |
 | withHeader | 控制是否显示 header 栏, 默认为 true, 当此项为 false 时, title attribute 和 title slot 均不生效 | boolean | - | true |
 | modal-class | 遮罩层的自定义类名 | string | - | - |
+| z-index | 设置 z-index | number | - | - |
```
Comment (YunYouJun) [inline website/docs/zh-CN/drawer.md:438]: Just add a blank after this.

File: website/docs/jp/drawer.md [hunk 2]
```diff
@@ -437,6 +441,7 @@ Drawerは `destroyOnClose` というAPIを提供しています。これはフ
 | title | Drawerのタイトルは、スロットの名前を指定して設定することもできます。 | string | — | — |
 | withHeader | デフォルトは true で、withHeader が false に設定されている場合は `title attribute` と `title slot` の両方が動作しません。 | boolean | - | true |
 | modal-class | シャドウレイヤーの追加クラス名 | string | — | — |
+| z-index | set z-index | number | - | - |
```
Comment (YunYouJun) [inline website/docs/jp/drawer.md:444]: Too.

**Review by YunYouJun — APPROVED** [review]

**Comment by YunYouJun** [comment]
Thanks.

**Comment by Otto-J** [comment]
😁

**Pull request merged**